tests/integration/test_bayesian_optimization.py

```diff
@@ -180,6 +181,13 @@ def OPTIMIZER_PARAMS() -> Tuple[
                     num_query_points=3,
                 ),
             ),
+            (
+                15,
+                EfficientGlobalOptimization(
+                    GreedyContinuousThompsonSampling(),
```

Comment:
Have you checked that alltests pass (either running locally or kicking off a manual job on GitHub)?

Reply:
yeep